BUILDING.md

```diff
@@ -600,9 +600,9 @@ $ make
 [Intl](https://github.com/nodejs/node/blob/master/doc/api/intl.md) support is
 enabled by default, with English data only.
```

Comment:
Should this also be updated?

Reply:
And also "(The embedded small-icu included in the default Node.js source does not include all locales.)" below.

Reply:
Good catch. I think I've addressed this now.